node_exporter.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
+	promcollectors "github.com/prometheus/client_golang/prometheus/collectors"
```

Comment:
did we make a breaking change in client_golang without bumping to v2?

Reply:
oh, just deprecated

Reply:
Yup, moved some functions to a new location. But the name conflicts with some node_exporter internals.